src/lib/core/CHIPConfig.h

```diff
@@ -1206,6 +1206,27 @@ extern const char CHIP_NON_PRODUCTION_MARKER[];
     "Please enable at least one of CHIP_CONFIG_EXAMPLE_ACCESS_CONTROL_FAST_COPY_SUPPORT or CHIP_CONFIG_EXAMPLE_ACCESS_CONTROL_FLEXIBLE_COPY_SUPPORT"
 #endif
 
+/**
+ * @def CHIP_CONFIG_ACCESS_RESTRICTION_MAX_ENTRIES_PER_FABRIC
+ *
+ * Defines the maximum number of access restriction list entries per
+ * fabric in the access control code's ARL attribute.
+ */
+#ifndef CHIP_CONFIG_ACCESS_RESTRICTION_MAX_ENTRIES_PER_FABRIC
+#define CHIP_CONFIG_ACCESS_RESTRICTION_MAX_ENTRIES_PER_FABRIC 10
```

Comment:
I don't see floors for these constants defined in the spec.  Or ceilings.  There should probably be such definitions.

Reply:
Tracking with https://github.com/project-chip/connectedhomeip/issues/35165